Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -296,16 +296,17 @@ function CompareSonarQubeVersionWith52
 #
 function GetOrFetchSonarQubeVersionString
 {         
-    $versionString = GetTaskContextVariable "MSBuild.SonarQube.ServerVersion"
+    $versionString = GetTaskContextVariable "MSBuild.SonarQube.Internal.ServerVersion"
```

Comment:
I accidentally commited some PR fixes to this branch instead of the base one. I'll point out the changes. This one is showing task extenders they should not rely on these variables.